app/src/main/resources/io/apicurio/registry/storage/impl/sql/postgresql.ddl

```diff
@@ -22,17 +22,20 @@ CREATE TABLE rules (tenantId VARCHAR(128) NOT NULL, groupId VARCHAR(512) NOT NUL
 ALTER TABLE rules ADD PRIMARY KEY (tenantId, groupId, artifactId, type);
 ALTER TABLE rules ADD CONSTRAINT FK_rules_1 FOREIGN KEY (tenantId, groupId, artifactId) REFERENCES artifacts(tenantId, groupId, artifactId);
 
-CREATE TABLE content (contentId BIGINT NOT NULL, canonicalHash VARCHAR(64) NOT NULL, contentHash VARCHAR(64) NOT NULL, content BYTEA NOT NULL);
-ALTER TABLE content ADD PRIMARY KEY (contentId);
-ALTER TABLE content ADD CONSTRAINT UNQ_content_1 UNIQUE (contentHash);
-CREATE INDEX IDX_content_1 ON content USING HASH (canonicalHash);
-CREATE INDEX IDX_content_2 ON content USING HASH (contentHash);
+CREATE TABLE content (contentHash VARCHAR(64) NOT NULL, content BYTEA NOT NULL);
+ALTER TABLE content ADD PRIMARY KEY (contentHash);
+
+CREATE TABLE tenant_content (tenantid VARCHAR(128) NOT NULL, contentid BIGINT NOT NULL, canonicalHash VARCHAR(64) NOT NULL, contenthash VARCHAR(64) NOT NULL);
+ALTER TABLE tenant_content ADD PRIMARY KEY (tenantid, contentid);
+ALTER TABLE tenant_content ADD CONSTRAINT FK_tenant_content_1 FOREIGN KEY (contenthash) REFERENCES content(contenthash);
+CREATE INDEX IDX_tenant_content_1 ON content USING HASH (canonicalHash);
+CREATE INDEX IDX_tenant_content_2 ON tenant_content USING HASH (contentHash);
```

Comment:
I want to raise this issue one more time for the @Apicurio/developers team - on balance do you think we should bother with the `tenant_content` table given the impact relative to the likely gain?  It makes the changes more substantial and makes the upgrade path harder.  Is it worth it?  Or should we just stick to the new `sequences` table change only (for both contentId and globalId)?